PS4k/src/com/vitco/layout/content/menu/SelectBarLogic.java

```diff
@@ -170,42 +171,31 @@ public boolean getStatus() {
         });
 
         // select all, expand selection, move to new layer, recolor
-        actionGroupManager.addAction("selection_interaction", "selection_tool_select_all", new StateActionPrototype() {
```

Comment:
I would move the abstract class above this line, so it's directly where it's used.